src/ui/components/TooltipMenu/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+.TooltipMenu {
+  border: 0;
+  border-radius: $border-radius-default;
+  width: 300px;
+}
+
+// This is the triangle at the top of the dropdown menu which is
+// actually just a tilted square.
+.TooltipMenu-arrow {
+  @include end(36px);
+
+  background-color: $white;
+  box-shadow: 0 0 2px transparentize($black, 0.5);
+  display: block;
+  height: 8px;
+  left: 50%;
+  position: absolute;
+  top: -4px;
+  transform: translateX(-50%) rotate(45deg);
+  width: 8px;
+  // Since we added a shadow, make sure the element appears underneath
+  // the Tooltip menu.
+  z-index: 1;
+}
+
+.TooltipMenu-inner {
+  border-radius: $border-radius-default;
+  box-shadow: 0 0 2px transparentize($black, 0.5);
+}
+
+.TooltipMenu-arrow-mask {
+  position: relative;
+  // Make sure this masks the Tooltip-arrow.
+  z-index: 2;
+}
+
+.TooltipMenu-ul {
+  background: $white;
+  border-radius: $border-radius-default;
+  list-style-type: none;
+  padding: 6px;
+  white-space: nowrap;
+
+  .ListItem {
+    color: $blue-50;
+    overflow: hidden;
+    padding: 0 6px 6px;
+    text-overflow: ellipsis;
+
+    &:first-child {
+      padding-top: 6px;
+    }
+
+    a:link,
+    a:visited,
+    a:hover,
+    a:active {
+      font-weight: normal;
+      text-decoration: none;
+    }
```

Comment:
Why don't we have black links like the dropdown menu? It should be blue only when `:active`, `:focus`, or `:hover`.

Reply:
Good idea, thanks. I actually began this patch by simply extending the `DropdownMenu` but then I realized edge detection is something I didn't want to write from scratch.